src/DFC_BusinessOntology.owl

```diff
@@ -761,235 +797,706 @@
         <DataProperty IRI="#batchNumber"/>
     </Declaration>
     <Declaration>
-        <DataProperty IRI="#bestBeforeDate"/>
+        <DataProperty IRI="#bestBeforeDate"/>
+    </Declaration>
+    <Declaration>
+        <DataProperty IRI="#brand"/>
+    </Declaration>
+    <Declaration>
+        <DataProperty IRI="#city"/>
+    </Declaration>
+    <Declaration>
+        <DataProperty IRI="#claim"/>
```

Comment:
@Alcoz it was these (lines 800 - 1271) I was referring to.

Reply:
Finally, the declaration came back to me as well and I don't know how to handle that. I chose to push them because the other logistic properties that I forgot were more important. Next time, we have to figured out what to do with that